packages/css/src/components/header/header.scss

```diff
@@ -13,29 +13,40 @@
   * Display grid is used to let both branding sections overlap.
   */
   display: grid;
+  font-family: var(--ams-header-font-family);
   padding-block: var(--ams-header-padding-block);
   padding-inline: var(--ams-header-padding-inline);
 }
 
-.ams-header__branding {
+.ams-header__logo-link {
   align-items: center;
   align-self: start; // To align the branding section to the top of the header when it wraps
```

Comment:
We no longer have a branding section. We could either restore it around the link to fix its layout, or change the comments mentioning it.